vanilla-framework/vanilla-framework
# Change name of 'aside' component to 'table of contents'

## Done

Renamed the "aside" component "table of contents" and updated the docs and example

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/table-of-contents/
- See the name change
- `cd docs`
- `./run`
- http://0.0.0.0:8104/en/patterns/table-of-contents
- See that the content and name are updated

## Details

Fixes #1827
